### ISSUE TITLE
weForms reCAPTCHA field is not working

### ISSUE THREAD

**Issue by mmarj** [weDevsOfficial/weforms#187]
**Describe the bug**: 
weForms reCAPTCHA field is not working.
If you are using a reCAPTCHA field alongside your forms then during submitting the data it will not let you submit the data after checking the validation. Will give the message "reCAPTCHA validation failed". But with the same reCAPTCHA API key, it will work with another plugin like WPUF.

**How to Reproduce**: 
Visit WP-admin > weForms > All forms > create a new form with the reCAPTCHA field. Published the form through a page. Then try to submit the form after validating the Captcha, you will catch an error message (reCAPTCHA validation failed) on the browser.

**Expected behavior**: 
Though the reCAPTCHA field is validated and checked then it should let the users submit the data.

**Additional input (Screenshot/video)**: 
Please check the video for a clear output result.

[A quick video](https://drive.google.com/file/d/1c0laAblyWRb5jo4in_C1Boj6nAJI9GNX/view?usp=sharing)
Feel free to let me know if you may need further information.

Thank you.

**Comment by jjones419** [comment]
I am having the same issue.  Not to mention before I started getting the failed validation errors continuously I was having issues when using reCaptcha v2 failing validation sometimes and blocking ip address through my hosting company.

**Comment by jjones419** [comment]
Since then I have switched to using reCaptcha v3 and generated new keys and I am still getting the "validation failed" errors that you mentioned above.

**Comment by bompus** [comment]
I don't believe weForms works yet with v3, but it would be nice if that can be added. However, I was able to pinpoint a typo that was recently introduced and this fixes the issue for me. Reference my commit above to see the one-liner change needed while you wait for the fix in the next release.

**Comment by MDAdler** [comment]
I have a site that has been running weForms with reCaptcha 2 for about three months. Updated to 1.4.3 on Jan 11 and no problems. Updated to 1.4.4 on Jan 17. weForm with reCaptcha still working. Went to add a new contact page with new contact form 4 hours ago and lots of things went wrong including reCaptcha returning a false "reCAPTCHA validation failed".

I have not checked for threads on the other issues but will included them here for completeness.

"Submit" button text was reset to "Submit Query". Any attempt to change the Submit button test shows in the preview as being successful but reloading the form shows that the change to the text does not stick.

Any attempt to change the default response text - "Thanks for contacting us! We will get in touch with you shortly." also fails to stick have been change. Previous changes to this text have been reset to the default.

Basically, any changes you attempt to make on the settings tab will not stick.

Deactivating the plugin and deleting it should delete all the data associated with the plugin. You even get a warning pop-up to remind you that all the data associated with the plugin will be lost (e.g. form definitions, form submission logs, etc.) but it is not. Reinstalling the plugin revives all the old and corrupted forms.
